cmd/mungedocs/toc.go

```diff
@@ -77,8 +77,12 @@ func buildTOC(markdown []byte) ([]byte, error) {
 			indent := strings.Repeat("  ", numSharps-1)
 			bookmark := strings.Replace(strings.ToLower(heading), " ", "-", -1)
 			// remove symbols (except for -) in bookmarks
-			r := regexp.MustCompile("[^A-Za-z0-9-]")
-			bookmark = r.ReplaceAllString(bookmark, "")
+			stripSpecialChars := regexp.MustCompile("[^A-Za-z0-9-]")
+			bookmark = stripSpecialChars.ReplaceAllString(bookmark, "")
+
+			collapseDashes := regexp.MustCompile("-+")
```

Comment:
Another way to do this is to do `stripSpecialChars.ReplaceAllString(...)` prior to `strings.Replace(..., "", "-", -1)`.  Which points out another bug, I think.  It should collapse multiple consecutive spaces into one "-".  I think it doesn't do that right now.